packages/core/src/artifacts.ts

```diff
@@ -4,7 +4,7 @@ import { createRequire } from 'node:module'
 import { printSchema, GraphQLSchema } from 'graphql'
 import { getGenerators, formatSchema } from '@prisma/internals'
 import type { KeystoneConfig } from './types'
-import { printGeneratedTypes } from './lib/schema-type-printer'
+import { printGeneratedTypes } from './lib/typescript-schema-printer'
```

Comment:
Renamed to align with `prisma-schema-printer`, I don't know why it was `typescript-schema-printer.tsx` though...